modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/roundstartslime.dm

```diff
@@ -39,33 +57,53 @@
 	button_icon_state = "alter_form"
 	icon_icon = 'modular_skyrat/master_files/icons/mob/actions/actions_slime.dmi'
 	background_icon_state = "bg_alien"
+	/// Do you need to be a slime-person to use this ability?
 	var/slime_restricted = TRUE
 
-/datum/action/innate/slime_change/admin
+/datum/action/innate/slime_change/unrestricted
 	slime_restricted = FALSE
 
 /datum/action/innate/slime_change/Activate()
-	var/mob/living/carbon/human/H = owner
-	if(slime_restricted && !isjellyperson(H))
+	var/mob/living/carbon/human/alterer = owner
+	if(slime_restricted && !isjellyperson(alterer))
 		return
-	if(slime_restricted)
-		H.visible_message(span_notice("[owner] gains a look of \
-		concentration while standing perfectly still.\
-		 Their body seems to shift and starts getting more goo-like."),
-		span_notice("You focus intently on altering your body while \
-		standing perfectly still..."))
+	if(isjellyperson(alterer))
+		alterer.visible_message(span_notice("[owner] gains a look of concentration while standing perfectly still. \
+		Their body seems to shift and starts getting more goo-like."),
+		span_notice("You focus intently on altering your body while standing perfectly still..."))
```

Comment:
I don't see a point in the isjellyperson, in the original code and in the modified.

Reply:
I think the idea is that you don't become goo-like if you're not a slime... But the way it was checked originally was weird and I was very torn on leaving the check in here or not. I'll probably just remove it.